boards/arm/nrf9160_innblue22/doc/index.rst

```diff
@@ -0,0 +1,145 @@
+.. _nrf9160_innblue22:
+
+nRF9160 INNBLUE22
+#################
+
+Overview
+********
+
+The nRF9160 innblue22 is a cellular IoT sensor development board, which
+is based on the nRF9160 SiP, and features NB-IoT and LTE-M connectivity.
```

Comment:
It would be nice if this included information about who makes it and a product information page, so interested parties can get one.

If this has not been released, please add that as soon as it's available.  (google isn't finding anything relevant about this device.)

Reply:
We are still busy with the info page and marketing... I will update as soon as it will be ready. Last few weeks were a real slowdown...
thank you for your feedback!